### ISSUE TITLE
Exporting to Matplotlib window fails using Python 3

### ISSUE THREAD

**Issue by jburgalat** [pyqtgraph/pyqtgraph#1056]
Hi all, 

Using python 3, pyqtgraph cannot export plots to Matplotlib window.

### Code to reproduce
The exception will be triggered using any of the examples of pyqtgraph. Just try to export the plot to Matplotlib window.

### Expected behavior
 A matplotlib window, I guess. I was just curious and tried the action.

### Real behavior
A crash related to a the following exception. One just have to replace `iteritems` method by `items` to get rid of the exception (or else define an alias for iteritems on if using python 3 ?). I did not have tested it myself so I cannot say if some other errors can occur in the process.

```
Traceback (most recent call last):
  File "F:\APPLICATIONS\Anaconda3\lib\site-packages\pyqtgraph\GraphicsScene\exportDialog.py", line 132, in exportClicked
    self.currentExporter.export()
  File "F:\APPLICATIONS\Anaconda3\lib\site-packages\pyqtgraph\exporters\Matplotlib.py", line 74, in export
    self.cleanAxes(ax)
  File "F:\APPLICATIONS\Anaconda3\lib\site-packages\pyqtgraph\exporters\Matplotlib.py", line 46, in cleanAxes
    for loc, spine in ax.spines.iteritems():
AttributeError: 'collections.OrderedDict' object has no attribute 'iteritems'
```

### Tested environment(s)

 * PyQtGraph version: 0.10.0
 * Qt Python binding: PyQt5 5.9.2 Qt 5.9.6
 * Python version: 3.7.3
 * NumPy version: 1.16.2
 * Operating system: Windows
 * Installation method: pip

**Comment by jburgalat** [comment]
👍 It works like a charm with the develop version ! 
I guess we can close this issue ? Or would you like to leave it opened until the next release ?

**Comment by j9ac9k** [comment]
@jburgalat thanks for verifying, I'm good with the issue being closed.

Thanks for following up!  

Also for the record, I'm using pyqtgraph from the develop branch in my work and personal projects, I know it's less than ideal, but I do consider this project stable enough.  Hopefully we'll get the new stable release out sooner than later.

**Comment by ksunden** [comment]
We have now released 0.11.0rc0 to [pypi](https://pypi.org/project/pyqtgraph/0.11.0rc0/).
